src/main/java/com/github/blindpirate/gogradle/crossplatform/DefaultGoBinaryManager.java

```diff
@@ -19,12 +19,7 @@
 
 import com.github.blindpirate.gogradle.GolangPluginSetting;
 import com.github.blindpirate.gogradle.core.cache.GlobalCacheManager;
-import com.github.blindpirate.gogradle.util.Assert;
-import com.github.blindpirate.gogradle.util.CompressUtils;
-import com.github.blindpirate.gogradle.util.HttpUtils;
-import com.github.blindpirate.gogradle.util.IOUtils;
-import com.github.blindpirate.gogradle.util.ProcessUtils;
-import com.github.blindpirate.gogradle.util.StringUtils;
+import com.github.blindpirate.gogradle.util.*;
```

Comment:
It seems that '*' violates one of checkstyle rules. When I ran `./gradlew check`, I got the following error report:

```
Error Description | Line
Using the '.*' form of import should be avoided - com.github.blindpirate.gogradle.util.*. | 22
```